<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widget{
	background-color:#F9F9F9;
    border-radius: 6px;
}</string>
     </property>
     <widget class="QPushButton" name="pushButton_close">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_close{border-image: url(:/images/icon-close-down.png);}
QPushButton#pushButton_close:pressed{border-image: url(:/images/icon-close-up.png);}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>201</y>
        <width>218</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_username">
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_username{
	background-color:rgba(255, 255, 255, 0);
}</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>输入账号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>1</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>284</y>
        <width>216</width>
        <height>1</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>216</width>
        <height>1</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>216</width>
        <height>1</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>160</y>
        <width>218</width>
        <height>43</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_repoUrl">
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_repoUrl{
	background-color:rgba(255, 255, 255, 0);
}</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>输入远程仓库地址</string>
         </property>
         <property name="cursorMoveStyle">
          <enum>Qt::LogicalMoveStyle</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>216</width>
           <height>1</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>216</width>
           <height>1</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>243</y>
        <width>218</width>
        <height>42</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit_password">
         <property name="minimumSize">
          <size>
           <width>196</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>196</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLineEdit#lineEdit_password{
	background-color:rgba(255, 255, 255, 0);
}</string>
         </property>
         <property name="frame">
          <bool>false</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>输入密码</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_enter">
         <property name="minimumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#pushButton_enter{border-image: url(:/images/icon-enter-down.png);}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_initLocal">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>330</y>
        <width>100</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#pushButton_initLocal{
	color:#9B9B9B;
}</string>
      </property>
      <property name="text">
       <string>本地使用</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="pixmap">
       <pixmap resource="../resources/resources.qrc">:/images/icon-logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="indent">
       <number>0</number>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <zorder>layoutWidget</zorder>
     <zorder>pushButton_close</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>layoutWidget</zorder>
     <zorder>frame_6</zorder>
     <zorder>pushButton_initLocal</zorder>
     <zorder>label</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
